sandy/core/endf6.py

```diff
@@ -271,17 +271,11 @@ def get_endf6_file(library, kind, zam, to_file=False):
         text = "".join([foo_read(name, url) for name in files.values()])
         tape = Endf6.from_text(text)
     else:
-        if type(zam) == pd.Series:
-            zam = zam.values()
-        if zam.__len__() != 1:
-            filename = files[zam[0]]
-            tape = foo_get(filename, url)
-            for nuclide in zam[1:]:
-                filename = files[nuclide]
-                tape = tape.merge(foo_get(filename, url))
+        if hasattr(zam, "__len__"):
```

Comment:
Last comment, this section is better like this, right?!

Yours might fail in some specific case!

```
    if hasattr(zam, "__len__"):
        tapes = map(lambda x: foo_get(files[x], url), zam)
        tape = reduce(lambda x, y: x.add_sections(y.data), tapes)
    else:
        if str(zam).lower() == 'all':
            if kind.lower() == 'xs':
                raise ValueError("'all' option is not available for xs")
            text = "".join([foo_read(name, url) for name in files.values()])
            tape = Endf6.from_text(text)
        else:
            tape = foo_get(files[zam], url)
```

Reply:
can you check this?

Reply:
In this order we have a problem: str variables has the atributte __len__

Reply:
So the program fail

Reply:
ok, then keep it as it is!